src/api/app/models/package.rb

```diff
@@ -891,6 +891,28 @@ def buildresults(prj = project)
     local_build_results
   end
 
+  def jobhistorylist(project, repository, arch, limit = 100)
```

Comment:
Nitpicking, but the rubyesque way name it would be something like jobhistory_list, right?:-)